docs/user-guide/advanced-configuration/assigning-awx-pods-to-specific-nodes.md

```diff
@@ -12,7 +12,7 @@ example, specifying `task_tolerations` will allow the AWX task pod to be schedul
 | Name                             | Description                              | Default  |
 | -------------------------------- | ---------------------------------------- | -------  |
 | postgres_image                   | Path of the image to pull                | postgres |
-| postgres_image_version           | Image version to pull                    | 13       |
+| postgres_image_version           | Image version to pull                    | 15       |
```

Comment:
Should be `quay.io/sclorg/postgresql-15-c9s` and `latest`.

Alternatively, the lines `postgres_image` and `postgres_image_version` may be deleted since there is little point in having these information in this table.

Reply:
Resolved, thanks!